src/dualize.jl

```diff
@@ -50,4 +46,30 @@ function dualize(primal_model::MOI.ModelLike, dual_problem::DualProblem{T}, dual
                                   primal_objective, con_types)
 
     return dual_problem
-end
\ No newline at end of file
+end
+
+# JuMP dualize
+function dualize(model::JuMP.Model; factory::Union{OptimizerFactory, Nothing} = nothing, dual_names::DualNames = DualNames("", ""))
+    # Create an empty JuMP model
+    JuMP_model = JuMP.Model()
+
+    if model.moi_backend.mode != MOIU.AUTOMATIC # Only works in AUTOMATIC mode
```

Comment:
I think `MOIU.mode(model)` works

Reply:
The problem is that `mode` is not defined when in `DIRECT_MODE` so  `MOIU.mode(model)` returns a method error.

Reply:
`model.moi_backend.mode` won't work either in direct mode

Reply:
It's `JuMP.mode(model)`, this works even in direct mode

Reply:
`JuMP.mode(model) != JuMP.AUTOMATIC` should work

Reply:
Yes :) But very weird that it was passing the tests